client/src/rpc_request.rs

```diff
@@ -111,6 +111,7 @@ pub enum RpcRequest {
     SendTransaction,
     SimulateTransaction,
     SignVote,
+    GetStakeMinimumDelegation,
```

Comment:
Can you insert this in A-Z order, please? (The indexes of these variants aren't used for anything.)